clang/lib/CIR/CodeGen/CIRGenDecl.cpp

```diff
@@ -480,6 +480,8 @@ CIRGenModule::getOrCreateStaticVarDecl(const VarDecl &D,
       getModule(), getLoc(D.getLocation()), Name, LTy, false, Linkage);
   // TODO(cir): infer visibility from linkage in global op builder.
   GV.setVisibility(getMLIRVisibilityFromCIRLinkage(Linkage));
+  GV.setVisibilityAttrAttr(
```

Comment:
From my previous comment: `setVisibility` and `setVisibilityAttrAttr` are similar and confusing.